src/style.css

```diff
@@ -1,3 +1,85 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url('Roboto-Medium.ttf');
+}
+
+html {
+  --brand-color: #00acdc;
+
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100dvw;
+  height: 60px;
```

Comment:
you don't need to set height for header directly, set height property for elements instead

Reply:
Have changed this, now we are using height for links instead of header.